app/models/event_schedule.rb

```diff
@@ -37,6 +39,16 @@ def replacement?
 
   private
 
+  def start_after_end_hour
+    errors.
+    add(:start_time, "can't be after the conference end hour (#{event.program.conference.end_hour})") if event.program && event.program.conference && event.program.conference.end_hour && start_time && start_time.strftime('%H').to_i > event.program.conference.end_hour
```

Comment:
@lagartoflojo @ChrisBr Yes we do have it. Sorry my bad,I was focusing on the test cases. I have made the changes. :+1: